Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -213,6 +215,50 @@ public final class CoreDataManager: StorageManagerType {
             return debugMessages
         }
     }
+
+    /// Note that we have to enumerate all the files in our sandbox to get these properties, so this takes some time.
+    /// It is intended _only_ for use when logging a crash, or other high-value log which won't get in the user's way.
+    private static func storageSizeLogProperties() -> [String: Int64]? {
+        let directoryUrls = [
+            "Bundle": Bundle.main.bundleURL,
+            "Documents": FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!,
+            "Library": FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first!,
+            "Temp": FileManager.default.temporaryDirectory,
+        ]
+
+        let directorySizes = directoryUrls.mapValues { url in
+            getAllocatedStorageSize(for: url)
+        }
+
+        let totalSize = directorySizes.values.reduce(0) { partialResult, size in
+            return partialResult + size
+        }
+
+        return directorySizes.merging(["Total": totalSize]) { first, second in
```

Comment:
nit; to avoid any warnings for unused variable

```suggestion
        return directorySizes.merging(["Total": totalSize]) { first, _ in
```